Kitodo/src/main/java/org/goobi/io/SafeFile.java

```diff
@@ -315,4 +315,8 @@ public URI toURI() {
     public URL toURL() throws MalformedURLException {
         return delegate.toURL();
     }
+
+    public File getDelegate() {
```

Comment:
Moved `start` method does not use any of the `list` or `listFiles` methods from `java.io.File` which are known returning `null` values on errors instead of throwing exceptions. `SafeFile` class was introduced to wrap this `null` values to empty list return values to get ride of ugly not null checks every where a `list` or `listFiles` methods are used.